tests/gold_tests/headers/replays/normalized_ae_varied_transactions.replay.yaml

```diff
@@ -784,11 +812,6 @@ sessions:
         - [ X-Response-Identifier, { value: Br-Accept-Encoding, as: equal } ]
         - [ X-Cache, { value: hit-fresh, as: equal } ]
 
-  # NOTICE: This case should make Accept-Encoding header br, gzip;q=0.8 match
-  #         the alternate of br, gzip Accept-Encoding header.
-  #         However, due to the implementation of calculate_quality_of_match(),
-  #         ATS matches the alternate of gzip Accept-Encoding header.
-  #         The result is DIFFERENT from the description of proxy.config.http.normalize_ae: 3
```

Comment:
Can you help me understand why this comment is being removed? Is the normalize_ae 3 alternate matching behavior changing?

Tagging @maskit / @moonchen since I think they worked with this test in the past.

Reply:
that is correct - i have fixed this bug whilst working on adding zstd to the normalise accept-encoding implementation